h2o-docs/src/product/explainability.rst

```diff
@@ -0,0 +1,205 @@
+Model Explainability
+====================
+
+H2O offers a convenient interface to many explainabilty methods in H2O.  The functions work for individual models, as well a list of models or an `H2O AutoML object <automl.html>`__
+
+
+Explainability Interface
+------------------------
+
+The H2O explainability interface is designed to be automatic -- all of the explainability feedback is generated with a single function, ``h2o.explain()``, which takes an object (one of the following: an H2O model, a list of models, an H2OAutoML object, an H2OAutoML Leaderboard slice), and a test frame.
+
+
+.. tabs::
+   .. code-tab:: r R
+
+        # Explain a model
+        exm <- h2o.explain(model, test)
+
+        # Explain an AutoML object
+        exa <- h2o.explain(aml, test)
+
+
+   .. code-tab:: python
+
+        # Explain a model
+        exm = model.explain(test)
+
+        # Explain an AutoML object
+        exa = aml.explain(test)
+
+
+
+
+Parameters
+~~~~~~~~~~
+
+TO DO: This was created looking at the R interface, may need some updates for Python.  
+
+- **object**: (R only) One of the following: an H2O model, a list of models, an H2OAutoML object, or an H2OAutoML leaderboard slice.
+
+- **test_frame**: An H2OFrame of test data.
+
+- **columns_of_interest**: A vector of column names. If specified, create plots only with these columns (where applicable).
+
+- **include_explanations**: If specified, do only the specified explanations. Mutually exclusive with exclude_explanations.
+
+- **exclude_explanations**: Exclude specified explanations.
+
+The available options (explainations) for ``include_explanations`` and ``exclude_explainations`` are:
+    
+    - ``"leaderboard"``
+    - ``"confusion_matrix"``
+    - ``"residual_analysis"``
+    - ``"variable_importance"``
+    - ``"variable_importance_heatmap"``
+    - ``"model_correlation_heatmap"``
+    - ``"shap_summary"``
+    - ``"pdp"``
+    - ``"ice"``
+
+- **top_n_features**: If ``columns_of_interest`` is missing, create plots only with the top n columns (where applicable).  Defaults to 5.
+
+- **best_of_family**: If True, explain only best of family models.
```

Comment:
This parameter just got removed to make the R version behaviour same as python's:

When explaining AutoML, show just best of family to get some general overview, but when explaining a list of models assume that the user actually handpicked the models and actually wants to see them all.

(still overridable by using the `plot_overrides`)